Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -14,28 +14,35 @@ public struct TuistCommand: AsyncParsableCommand {
         CommandConfiguration(
             commandName: "tuist",
             abstract: "Generate, build and test your Xcode projects.",
-            subcommands: [
-                BuildCommand.self,
-                CleanCommand.self,
-                DumpCommand.self,
-                EditCommand.self,
-                InstallCommand.self,
-                GenerateCommand.self,
-                GraphCommand.self,
-                InitCommand.self,
-                MigrationCommand.self,
-                PluginCommand.self,
-                RunCommand.self,
-                ScaffoldCommand.self,
-                TestCommand.self,
-                AuthCommand.self,
-                SessionCommand.self,
-                LogoutCommand.self,
-                CleanCommand.self,
-                ProjectCommand.self,
-                OrganizationCommand.self,
-                AnalyticsCommand.self,
-                CacheCommand.self,
+            subcommands: [],
+            groupedSubcommands: [
+                CommandGroup(name: "Start", subcommands: [
+                    InitCommand.self,
+                ]),
+                CommandGroup(name: "Develop", subcommands: [
+                    BuildCommand.self,
+                    CleanCommand.self,
+                    DumpCommand.self,
+                    EditCommand.self,
+                    InstallCommand.self,
+                    GenerateCommand.self,
+                    GraphCommand.self,
+                    MigrationCommand.self,
+                    PluginCommand.self,
+                    RunCommand.self,
+                    ScaffoldCommand.self,
+                    TestCommand.self,
+                    CleanCommand.self,
+                    CacheCommand.self,
+                ]),
+                CommandGroup(name: "Server", subcommands: [
+                    ProjectCommand.self,
+                    OrganizationCommand.self,
+                    AuthCommand.self,
+                    SessionCommand.self,
+                    AnalyticsCommand.self,
```

Comment:
A bit unrelated but I wonder if we should remove this one since it only opens the server.

We could also replace it with a command similar to `gh` CLI:
`gh repo view tuist/tuist` -> `tuist project view tuist/tuist`

What do you think?

Reply:
Good idea! I pushed a `tuist project view` command to this PR :)